docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -113,11 +113,16 @@ For information about exceptions and how to use them, refer to
 [[signals-to-timelines]]
 ==== View alerts in Timeline
 
-To view an alert in Timeline, click the *Investigate in timeline* button in the Alerts table. Alternatively, select *Take action* -> *Investigate in timeline* in the alert details flyout.
+* To view a single alert in Timeline, click the *Investigate in timeline* button in the Alerts table. Alternatively, select *Take action* -> *Investigate in timeline* in the alert details flyout.
 
 [role="screenshot"]
 image::images/timeline-button.png[Investigate in timeline button, 300]
 
+* To view multiple alerts in Timeline (up to 2,000), select them using the checkboxes on the left of the Alerts table, then click *Selected N alerts*, where *N* is the number of alerts selected, and click *Investigate in timeline*.
```

Comment:
We probably don't need as much detail or explanation for a fairly common workflow, and we have a slightly different format for labels with variable text ([guideline here](https://wiki.elastic.co/pages/viewpage.action?pageId=105914797) in TL;DR section):

> Use _x_ (italicized) for variable text in UI component names, such as: "Click **Select all _x_ rules**, and then select **Activate**." 

```suggestion
* To view multiple alerts in Timeline (up to 2,000), select the checkboxes next to the alerts, then click *Selected _x_ alerts* -> *Investigate in timeline*.
```